bin/bios_mobo_versions.sh

```
@@ -1 +1,10 @@
-ansible -v -b -m shell -a 'echo $(hostname),$(lshw -json | jq ".children[0].product"),$(lshw -json | jq ".children[0].children[0].version"),$(lshw -json | jq ".children[0].serial")' all | grep -E -o '(cic|dnp)(ws|cs)[0-9]+,.*' | sort
+#ansible -v -b -m shell -a 'echo $(hostname),$(lshw -json | jq ".children[0].product"),$(lshw -json | jq ".children[0].children[0].version"),$(lshw -json | jq ".children[0].serial")' all | grep -E -o '(cic|dnp)(ws|cs)[0-9]+,.*' | sort
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)